script/github-release

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
```

Comment:
It would be, after it stabilizes in its API a bit. For now I've coded it up manually since I wanted to possibly reuse this script in other Go projects that might not have hub available— although, the script itself could download and install hub just for the purposes of creating a release :)